sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveQl.scala

```diff
@@ -557,8 +557,11 @@ https://cwiki.apache.org/confluence/display/Hive/Enhanced+Aggregation%2C+Cube%2C
             "TOK_TABLEPROPERTIES"),
           children)
       val (db, tableName) = extractDbNameTableName(tableNameParts)
-
-      CreateTableAsSelect(db, tableName, nodeToPlan(query), allowExisting != None, Some(node))
+      if (ignores.exists(_ != None)) {
+        CreateTableAsSelect(db, tableName, nodeToPlan(query), allowExisting != None, Some(node))
+      } else {
+        CreateTableAsSelect(db, tableName, nodeToPlan(query), allowExisting != None, None)
```

Comment:
@yhuai I just found that, if we rely on `SemanticAnalyzer` to extract default format even user doesn't specify serde and format (so `desc` is never `None`), then `hasStorageSpec` will be `false` only if the table already existed.

If so, even `hive.convertCTAS` is enabled, it will turn to use Hive default format instead of using Parquet as the default file format for CTAS statements. I think that is not we want?